go/runtime/history/indexer.go

```diff
@@ -230,6 +317,13 @@ func (bi *BlockIndexer) reindexTo(ctx context.Context, height int64) error {
 		lastHeight = lastRetainedHeight
 	}
 
+	bi.mu.Lock()
+	bi.status = StatusReindexing
+	bi.startHeight = lastHeight
+	bi.endHeight = height
+	bi.started = time.Now()
```

Comment:
Should we change this by extending the end height only? This way, we can estimate next reindex more accurately, plus in the end the user can see the whole range.